CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## Development
+
+### New Features
+
+* **Added ability to populate all `target` properties in the `puppetdb` plugin's `target_mapping`**
+  Previously the `target_mapping` field only supported populating target's
+  `uri`, `name` and `config` values. Additional fields are now able to be
+  pupulated on a target, a complete list is below:
+   * `alias`
+   * `config`
+   * `facts`
+   * `features`
+   * `name`
+   * `uri`
+   * `vars`
+
```

Comment:
We recently made changes to how we add release notes to the changelog. Instead of modifying the changelog directly, you can add a label (e.g. `!feature`) to a commit and put the changelog entry directly after it. More info here: https://github.com/puppetlabs/bolt/blob/master/.github/rulesets.yaml#L8-L24

Reply:
Applied this changelog to the commit message!